src/main/java/com/gitblit/utils/FileUtils.java

```diff
@@ -154,6 +155,14 @@ public static String readContent(File file, String lineEnding) {
 			System.err.println("Failed to read content of " + file.getAbsolutePath());
 			t.printStackTrace();
 		} finally {
+			if (reader != null){

```

Comment:
I don't believe closing the BufferedReader wrapper is necessary as the underlying stream is closed.